msal/build.gradle

```diff
@@ -105,15 +105,24 @@ dependencies {
     implementation "com.google.code.gson:gson:$rootProject.ext.gsonVersion"
     // test dependencies
     testImplementation "junit:junit:$rootProject.ext.junitVersion"
-    testImplementation "org.mockito:mockito-core:$rootProject.ext.mockitoCoreVersion"
+    // TODO the below dependency should be mockito-core but the MockMaker isn't working...
+    testImplementation "org.mockito:mockito-android:$rootProject.ext.mockitoAndroidVersion"
```

Comment:
`mockito-core` is throwing weird exceptions on my test device. I've changed this dependency to use `mockito-android` though it returns its own strange warnings (tests pass, however). This is something we should revisit, but we're unblocked for now.

```
IMPORTANT NOTE FROM MOCKITO:

You included the 'mockito-android' dependency in a non-Android environment.
The Android mock maker was disabled. You should only include the latter in your 'androidTestCompile' configuration
If disabling was a mistake, you can set the 'org.mockito.mock.android' property to 'true' to override this detection.

Visit https://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/Mockito.html#0.1 for more information

Process finished with exit code 0
```

Reply:
I'll open an issue for this and link back here.

Reply:
See #284